### PR TITLE
New package: cpuset-1.6pre1

### PR DESCRIPTION
**Pull request by pzel**

### CODE REVIEW

File: srcpkgs/cpuset/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pkgname=cpuset
+version=1.6pre1
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-devel"
+depends="python3 python3-future"
+short_desc="Wrapper to make kernel cpusets facilities easier to use"
+maintainer="Simon Zelazny <zelazny@mailbox.org>"
+license="GPL-2"
```
Comment (cr6git) [inline srcpkgs/cpuset/template]: `license="GPL-2.0-or-later"`

File: srcpkgs/cpuset/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'cpuset'
+pkgname=cpuset
+version=1.6pre1
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-devel"
+depends="python3 python3-future"
```
Comment (cr6git) [inline srcpkgs/cpuset/template]: remove `python3`

**Comment by pzel** [comment]
Thank you @cr6git, changes added.

File: srcpkgs/cpuset/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-devel"
+depends="python3-future"
+short_desc="Wrapper to make kernel cpusets facilities easier to use"
+maintainer="Simon Zelazny <zelazny@mailbox.org>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/lpechacek/cpuset"
+distfiles="https://github.com/lpechacek/${pkgname}/archive/v${version}.tar.gz"
+checksum="f05baaeef222265e12f6c890d19c841237fdc641ddcd21abf76b2a189d45a6ac"
+
```
Comment (maxice8) [inline srcpkgs/cpuset/template]: unnecessary blank line

**Pull request closed without merging**